src/Compilers/Core/Portable/Operations/ILocalFunctionOperation.cs

```diff
@@ -23,10 +23,18 @@ public interface ILocalFunctionOperation : IOperation
         /// Body of the local function.
         /// </summary>
         /// <remarks>
-        /// Expression-bodied local functions are normalized here to be a single <see cref="IOperation"/>
-        /// inside of a block statement.
+        /// This will return the <see cref="BlockBody"/> if it exists, and the <see cref="ExpressionBody"/> if the <see cref="BlockBody"/> does not exist.
+        /// If both exist, this will return just the <see cref="BlockBody"/>, and you must use <see cref="ExpressionBody"/> to retrieve the expression body.
```

Comment:
I chose to add two new properties here to mirror `IMethodBodyOperation`, and so that we could prevent a breaking change in the `Body` property by no longer returning anything for expression body form.

Reply:
@jinujoseph we should add this to the list for tomorrow's discussion.